api/src/importers/coors/admin-sanction-utils.js

```diff
@@ -0,0 +1,155 @@
+    adminSanction['recordName'] = (csvRow['case_no'] && `Case No. ${csvRow['case_no']}`) || '';
+
+    const ministry = 'Ministry of Forests, Lands and Natural Resource Operations'
+    adminSanction['issuingAgency'] = ministry;
+    adminSanction['author'] = ministry;
+    adminSanction['dateIssued'] = csvRow['effective_date'] || null;
+
+
+    const entityType = CsvUtils.getEntityType(csvRow) || null;
+    if (entityType === 'Company') {
+      adminSanction['issuedTo'] = {
+        type: 'Company',
+        companyName: csvRow['business_name'] || ''
+      };
+    }
+
+    if (entityType === 'Individual') {
+      adminSanction['issuedTo'] = {
+        type: 'Individual',
+        firstName: csvRow['first_name'] || '',
+        middleName: csvRow['middle_name'] || '',
+        lastName: csvRow['last_name'] || '',
+        dateOfBirth: csvRow['birth_date'] || null
+      };
+    }
+
+    adminSanction['location'] = csvRow['location_of_violation'] || '';
+
+    // Section 85 specific fields
+    if (csvRow['record_type_code'] === 'S85' && csvRow['business_reviewed_ind'] === 'Y') {
+      adminSanction['legislation'] = {
+        act: 'Wildlife Act',
+        regulation: '',
+        section: 85,
+        subSection: '',
+        paragraph: ''
+      };
+      adminSanction['legislationDescription'] = 'Angling, hunting and/or Limited Entry Hunting licence action for failure to pay fine';
+
+      adminSanction['penalties'] = [
+        {
+          type: 'Other',
+          penalty: {
+            type: 'Other',
+            value: null
+          },
+          description: 'Licences, LEH, Permits Cancelled'
+        }
+      ];
+      adminSanction['summary'] = this.buildSummary(csvRow, 85);
+    // Section 24 specific fields
+    } else if (csvRow['record_type_code'] === 'S24' && csvRow['business_reviewed_ind'] === 'Y') {
+      adminSanction['legislation'] = {
+        act: 'Wildlife Act',
+        regulation: '',
+        section: 24,
+        subSection: '',
+        paragraph: ''
+      };
+      adminSanction['legislationDescription'] = 'Angling, hunting, firearm and/or LEH licence action prompted by violations';
+
+      adminSanction['penalties'] = [
+        {
+          type: 'Other',
+          penalty: {
+            type: 'Other',
+            value: null
+          },
+          description: 'Suspension or cancellation of licence(s)'
+        }
+      ];
+      adminSanction['summary'] = this.buildSummary(csvRow, 24);
+    } else {
+      // row not reviewed by business, do not process
+      return null;
+    }
+
+    return adminSanction;
+  }
+
+  buildSummary(csvRow, sectionNum) {
+    let legislation;
+    if (!csvRow['regulation_description']) {
+      legislation = csvRow['act']
+    } else {
+      legislation = csvRow['regulation_description']
+    }
+
+    let enforcement = '';
+    if (csvRow['enforcement_licence_code'] === 'DDC') {
+      enforcement = "Director's decision to suspend licence";
```

Comment:
`enforcement` value is slightly different than the ticket.  Same for the `else if` block

Reply:
updated to match latest enforcement value